setup.py

```diff
@@ -16,6 +16,7 @@
 setup(
     name='spotify_dl',
     version=version,
+    python_requires='>=3',
```

Comment:
Can make it support both. there are still too many people using python2 as default.